libraries/AP_Scripting/examples/rtsp_video_relay.lua

```diff
@@ -0,0 +1,643 @@
+--[[
+   RTSP Video Relay example
+
+   this relays RTSP video between a camera and a media server such as mediamtx
+--]]
+
+-- language checker does not understand socket_wrapper()
+---@diagnostic disable: undefined-field
+
+-- allow assigning of sockets to nil on close
+---@diagnostic disable: cast-local-type
```

Comment:
Better to just disable per line rather than for the whole file. But if it is SOP to asign nil we should probably see if we can make the checker OK with it.

Reply:
I changed to a next-line diagnostic for this one, but left the one for socket_wrapper global as it is needed on so many lines